site/_posts/2019-10-08-deactivating-reactivating-deleting-user-fusionauth-python.md

```diff
@@ -205,72 +214,72 @@ Save and execute the `reactivate.py` application on your server. Upon success, y
 
-{% include _image.liquid src="/assets/img/blogs/terry-example-unlocked.png" alt="Terry is now unlocked" class="img-fluid mb-4 mt-2 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/terry-example-unlocked.png" alt="Terry is now unlocked" class="img-fluid" figure=false %}
 
-## How to Delete A User With FusionAuth-Client
+## How to delete a user with FusionAuth Python client library
 
-For our next trick, we will be deleting a test user from the server. If you filled out a lot of profile information on your regular test user, you might want to create an empty test user for this tutorial. Just be sure to grab their UserID from the **Actions-->Manage** section of FusionAuth.
+For our next trick, we will be deleting a test user from the server. If you filled out a lot of profile information on your regular test user, you might want to create an empty test user for this tutorial. Just be sure to grab their "User Id" from the "Manage User" section of FusionAuth.
 
-{% include _image.liquid src="/assets/img/blogs/delete-test-user.png" alt="A user to delete" class="img-fluid mb-4 mt-2 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/delete-test-user.png" alt="A user to delete" class="img-fluid" figure=false %}
 
-Pop open your Python editor, and create a new file called `deleteuser.py`. Add the FusionAuth_client Library, Your API key, and your server info.
+Pop open your Python editor, and create a new file called `delete.py`. Import the FusionAuth client library, your API key, and your server info.
 
 ```python
 from fusionauth.fusionauth_client import FusionAuthClient
 
-#  You must supply your API key and URL here
+# You must supply your API key and URL here
 client = FusionAuthClient('YOUR-API', 'http://localhost:9011')
 ```
 
-Now, add this section to the application, replacing the userID with your own:
+Now, add this section to the application, again replacing the `YOUR-USER-ID` with your test user's Id:
 
 ```python
-# Delete User For A Given ID
-client_response =client.delete_user('USER-ID')
+# Delete user for a given Id
+client_response =client.delete_user('YOUR-USER-ID')
 if client_response.was_successful():
-	print(client_response.success_response)
+  print(client_response.success_response)
 else:
-	print(client_response.error_response)
+  print(client_response.error_response)
 ```
 
-Save and execute your `deleteuser.py` application. (**Caution:** Executing this action will actually delete the user.)
+Save and execute your `delete.py` application. (**Caution:** Executing this action will actually delete the user.)
 
-If successful, you will get a response of `None` and the user will now be deleted from FusionAuth. You can test this again by refreshing the **Users** page in your browser.  
+If successful, you will get a response of `None` and the user will now be deleted from FusionAuth. You can test this again by refreshing the "Users" page in your browser.  
```

Comment:
Hmmm. Is this a hard delete or a soft delete? (Hard delete actually removes them from the system, soft delete lets you restore later.)  Looking at the source, it looks like a hard delete: https://github.com/FusionAuth/fusionauth-python-client/blob/master/src/main/python/fusionauth/fusionauth_client.py